src/components/Locations.jsx

```diff
@@ -0,0 +1,83 @@
+import { Location } from './Location';
+
+export function Locations() {
+  const locationsData = [
+    {
+      linkUrl: 'https://councildataproject.org/seattle',
+      imageUrl: 'imgs/seattle.jpg',
+      imageDescription: 'Seattle skyline',
+      title: 'Seattle, Washington',
+      metadata: 'Maintained by the CDP Core Team',
+    },
+    {
+      linkUrl: 'https://councildataproject.org/king-county',
+      imageUrl: 'imgs/king-county.png',
+      imageDescription:
+        'King County, Puget Sound, Lake Washington, and Mt. Rainier',
+      title: 'King County, Washington',
+      metadata: 'Maintained by the CDP Core Team',
+    },
+    {
+      linkUrl: 'https://councildataproject.org/portland',
+      imageUrl: 'imgs/portland.jpg',
+      imageDescription: 'Portland Downtown Waterfront',
+      title: 'Portland, Oregon',
+      metadata: 'Maintained by the CDP Core Team',
+    },
+    {
+      linkUrl: 'https://www.openmontana.org/missoula-council-data-project',
+      imageUrl: 'imgs/missoula.jpg',
+      imageDescription: 'Missoula Skyline',
+      title: 'Missoula, Montana',
+      metadata: 'Maintained by Open Montana',
+    },
+    {
+      linkUrl: 'https://councildataproject.org/denver',
+      imageUrl: 'imgs/denver.jpg',
+      imageDescription: 'Denver Skyline',
+      title: 'Denver, Colorado',
+      metadata: 'Maintained by the CDP Core Team',
+    },
+    {
+      linkUrl: 'https://councildataproject.org/alameda',
+      imageUrl: 'imgs/alameda.jpg',
+      imageDescription: 'Alameda Theatre in Night',
+      title: 'Alameda, California',
+      metadata: 'Maintained by the CDP Core Team',
+    },
+    {
+      linkUrl: 'https://councildataproject.org/boston',
+      imageUrl: 'imgs/boston.jpg',
+      imageDescription: 'Boston Skyline from the waterfront',
+      title: 'Boston, Massachusetts',
+      metadata: 'Maintained by the CDP Core Team',
+    },
+    {
+      linkUrl: '#',
```

Comment:
Can you update this link to point at the cookiecutter repo? I think the repo instead of the deployment issue. they may want to host it themselves.

Reply:
Great catch thank you!